tests/Unit/ParallelAlgorithms/LinearSolver/ConjugateGradient/Test_ConjugateGradientAlgorithm.cpp

```diff
@@ -25,49 +22,20 @@ struct SerialCg {
 };
 
 struct Metavariables {
```

Comment:
I tried that but haven't found a good way to do that because I'm mixing the aliases from the "serial" helpers, the "distributed" helpers and in future PRs also test-specific classes.